packages/manager/apps/hycu/src/data/api/orderCatalogHYCU.ts

```diff
@@ -0,0 +1,14 @@
+import apiClient from '@ovh-ux/manager-core-api';
+import { HYCUCatalog } from '@/types/orderCatalogHYCU.type';
+
+/**
+ * HYCU Catalog : Get HYCU Order Catalog
+ */
+export const getOrderCatalogHYCU = async (
+  ovhSubsidiary: string,
```

Comment:
```suggestion
  ovhSubsidiary: string,
```

To verify : But it's not : `CountryCode` type ? See: packages/manager/apps/hycu/src/hooks/guide/useGuideUtils.tsx

Reply:
Or : OvhSubsidiary